api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -228,5 +234,11 @@
 		<addForeignKeyConstraint constraintName="encounter_diagnosis_changed_by_fk"
 								 baseTableName="encounter_diagnosis" baseColumnNames="changed_by"
 								 referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="encounter_diagnosis_condition_coded_fk"
+								 baseTableName="encounter_diagnosis" baseColumnNames="condition_coded"
+								 referencedTableName="condition" referencedColumnNames="concept_id " />
```

Comment:
Are you sure the referenced table is condition?

Reply:
This was an error. It should be `concept`. Am changing it appropriately.